bin/index.js

```diff
@@ -170,23 +170,30 @@ updateLines();
 /*
  * Start speed test
  */
- 
+
 var options = {
   maxTime: 10000,
-  proxy : null,
-}
+  proxy: null,
+  sourceIp: null,
+};
 
 /*
  * get parameters
  */
 var errorParameter = null;
 process.argv.forEach(function (val, index, array) {
   if (index >= 2) {
-    if (val.startsWith("--proxy")) {
+    if (val.startsWith('--proxy')) {
```

Comment:
Could you change the `startsWith` to `val === '--proxy'`? Not sure why `--proxyficationatorial` should be considered valid. Ditto below. Also, would you mind changing the `!= undefined` to `!== undefined` (twice)?

Reply:
A late response, bit I fixed this..